From 1bed7c4730dcfebbb179c56e6d4c433c2a1429b8 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 4 Mar 2020 16:12:30 +0700
Subject: [PATCH 261/268] arm64: dts: renesas: hihope-common: add missing DU
 pins

This patch adds missing DU pins for common platform.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index b1a8943..5efec7b 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -384,6 +384,11 @@
 		groups = "i2c2_a";
 		function = "i2c2";
 	};
+
+	du_pins: du {
+		groups = "du_rgb888", "du_sync", "du_oddf", "du_clk_out_0";
+		function = "du";
+	};
 };
 
 &rcar_sound {
-- 
2.7.4

